lib/mission_control/jobs/engine.rb

```diff
@@ -11,17 +11,15 @@ class Engine < ::Rails::Engine
       isolate_namespace MissionControl::Jobs
 
       config.mission_control = ActiveSupport::OrderedOptions.new unless config.try(:mission_control)
-      config.mission_control.jobs = ActiveSupport::OrderedOptions.new
+      config.mission_control.jobs = MissionControl::Jobs
```

Comment:
We don't want the config to point to this module, we want `mission_control.jobs` to behave as any other app config. This should be kept as it was, as `ActiveSupport::OrderedOptions.new`.

Reply:
Makes sense, done